### PR TITLE
docker should run with non-root user

### PR DESCRIPTION
**Pull request by gloovers**
## Description
This change runs docker containers with non-root user. 
It will add more security to deployment and prevent of unintended leaks

## Testing
1. make infra
```
mixture of literal strings and interpolations. This deprecation applies only
to templates that consist entirely of a single interpolation sequence.


Apply complete! Resources: 2 added, 2 changed, 0 destroyed.
Releasing state lock. This may take a few moments...

Outputs:

bastion_instance_id = i-0bf4176b35c9f3037
cmd = {
  "tunnel" = {
    "down" = "ssh -S bastion.sock -O exit ubuntu@i-0bf4176b35c9f3037 "
    "status" = "ssh -S bastion.sock -O check ubuntu@i-0bf4176b35c9f3037"
    "up" = "ssh -M -S bastion.sock -fNT ubuntu@i-0bf4176b35c9f3037 "
  }
}
ssh_forward_config = [
  "# SSH over Session Manager",
  "host i-* mi-*",
  "ProxyCommand sh -c \"aws ssm start-session --target %h --document-name AWS-StartSSHSession --parameters 'portNumber=%p'\"\n",
]
[OK] Terraform output saved to ssm://dev/terraform-output
```
2. make <service_name>
```

Deploying new task definition...............................................................................................
Deployment successful

Deregister task definition revision
Successfully deregistered revision: 185
```
3. make env.debug
```
.....
=== AWS Environment Info ===
AWS_DEV_ENV_NAME: dev (User env is not configured)
AWS_ACCOUNT: XXXXXXXXXXXXXX
AWS_PROFILE: XXXXXXXXXXXXX
AWS_USER: XXXXXX
```

### CODE REVIEW

File: aws/main.mk [hunk 1]
```diff
@@ -15,8 +15,8 @@ AWS_ACCOUNT ?= $(shell [ -f ~/.aws/credentials ] && $(AWS) sts get-caller-identi
 AWS_DEV_ENV_NAME ?= $(shell [ -f ~/.aws/credentials ] && $(AWS) iam list-user-tags --user-name $(AWS_USER) | ( $(JQ) -e -r '.Tags[] | select(.Key == "devEnvironmentName").Value') || echo "$(ENV) (User env is not configured)")
 
 # $(AWS_ARGS) definition see in .infra/icmk/aws/localstack.mk
-AWS ?= $(DOCKER) run \
-	-v $(HOME)/.aws/:/root/.aws \
+AWS ?= $(DOCKER) run --user "$(CURRENT_USER_ID):$(CURRENT_USERGROUP_ID)" \
+	-v $(HOME)/.aws/:/.aws \
```
Comment (AutomationD) [inline aws/main.mk:19]: Are we mounting `.aws` to the root directory? Does this work with aws cli and standard SDK (for third-party tools like Terraform)?

**Review by AutomationD — CHANGES_REQUESTED** [review]
It sounds like we will need to test each of the commands to verify setting UID works.
We will also need to test it on multiple platforms.
Let's create some small test matrix that runs those commands somehow

File: aws/main.mk (hunk 1 above)
Comment (gloovers) [reply aws/main.mk:19]: yes, it works

**Comment by gloovers** [comment]
> 
> 
> It sounds like we will need to test each of the commands to verify setting UID works.
> We will also need to test it on multiple platforms.
> Let's create some small test matrix that runs those commands somehow

OK, I'm doing this

**Pull request closed without merging**